client/src/controllers/BreadcrumbsController.ts

```diff
@@ -0,0 +1,132 @@
+import { Controller } from '@hotwired/stimulus';
+
+export class BreadcrumbsController extends Controller {
+  static targets = ['w-breadcrumbs', 'toggle'];
```

Comment:
```suggestion
  static targets = [ 'toggle'];
```

You shouldn't need to create a target for the root element, aka the controlled element. This is already available as `this.element`.